modules/apps/journal/journal-service/src/main/java/com/liferay/journal/internal/content/compatibility/converter/JournalContentCompatibilityConverterImpl.java

```diff
@@ -220,7 +220,7 @@ private String _convertDDMFieldValue(
 	private String _convertLinkToLayoutValue(
 		Locale defaultLocale, String value) {
 
-		if (JSONUtil.isValid(value)) {
+		if (Validator.isNotNull(JSONUtil.createJSONObject(value))) {
```

Comment:
@slnn Is this SF's requirement to use `Validator.isNotNull`?

Reply:
No

Reply:
@dantewang 

Force pushed